frontend/README.md

```diff
@@ -0,0 +1,3 @@
+1.Copy .env.example to .env and set variables to a value
+
+2.Run npm run dev
```

Comment:
Write simple instruction to your project

Reply:
added